package.json

```diff
@@ -32,13 +41,29 @@
     "url": "https://github.com/bigbinary/neeto-hotkeys/issues"
   },
   "homepage": "https://github.com/bigbinary/neeto-hotkeys#readme",
+  "dependencies": {
+    "mousetrap": "^1.6.5",
+    "mousetrap-global-bind": "^1.1.0",
+    "platform": "^1.3.6",
+    "ramda": "^0.29.1",
+    "react": "^18.2.0"
+  },
   "devDependencies": {
     "@babel/eslint-parser": "^7.23.3",
     "@babel/preset-env": "^7.23.3",
     "@babel/preset-react": "^7.23.3",
+    "@bigbinary/babel-preset-neeto": "^1.0.7",
     "@bigbinary/eslint-plugin-neeto": "^1.1.31",
     "@bigbinary/neeto-cist": "^1.0.4",
     "@bigbinary/neeto-commons-frontend": "^2.1.38",
+    "@rollup/plugin-babel": "^6.0.4",
+    "@rollup/plugin-commonjs": "^25.0.7",
+    "@rollup/plugin-json": "^6.0.1",
+    "@rollup/plugin-node-resolve": "^15.2.3",
+    "@rollup/plugin-replace": "^5.0.5",
+    "@svgr/rollup": "^8.1.0",
+    "babel-plugin-preval": "^5.1.0",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
```

Comment:
Are you using these plugins anywhere?

Reply:
Yes. These are being used internally. If I remove them, I get errors.

<img width="1280" alt="Screenshot 2023-11-17 at 10 54 24 AM" src="https://github.com/bigbinary/neeto-hotkeys/assets/67572237/81c757f0-7c5b-410a-ac0a-1da22fc2c60a">

<img width="1280" alt="Screenshot 2023-11-17 at 10 53 11 AM" src="https://github.com/bigbinary/neeto-hotkeys/assets/67572237/3febb872-8ad4-4833-ac7b-d4bbcd79d33e">